CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Fixed
+- Resolve bug in `cert_file` parameter logic, in the `conjur` class, that attempts to
+  reassign immutable `$ssl_certificate` Puppet variable.
```

Comment:
the audience for this message is a user of this module; can you please revise this so that the message speaks to the _user impact_ of this change?

Reply:
Updated